esp_lcd_touch_xpt2046.c

```diff
@@ -141,6 +160,24 @@ static esp_err_t xpt2046_read_data(esp_lcd_touch_handle_t tp)
     uint32_t x = 0, y = 0;
     uint8_t point_count = 0;
 
+#ifdef CONFIG_XPT2046_INTERRUPT_MODE
+    if (tp->config.int_gpio_num != GPIO_NUM_NC)
+    {
+        // Check the PENIRQ pin to see if there is a touch
+        if (!gpio_get_level(tp->config.int_gpio_num))
```

Comment:
This check seems to be inverted from what `XPT2046_INTERRUPT_MODE` seems to imply (low = touched)

Reply:
You are correct. Fixed with a force push.